### PR TITLE
apps sc: upgrade harbor fix rook bug

### LINKED ISSUE #173: [2] Upgrade harbor helmchart to v1.6 when released to fix rook bug
When using harbor together with rook there is a potential bug that appears if the database pod is killed and restarted on a new node. The issue is reported here: https://github.com/goharbor/harbor-helm/issues/444
There is a fix in the helm chart but it has not been released yet, so when the next release comes, we should upgrade.

Acceptance criteria:
- Harbor helm chart is upgraded to a version where the described bug does not occur.
- Some test is done to check if the problem still persists.

### PR DESCRIPTION
**Pull request by robinelastisys**
**What this PR does / why we need it**:
When using harbor together with rook there is a potential bug that appears if the database pod is killed and restarted on a new node. This is fixed by upgrading Harbor to version 1.6.1.

**Which issue this PR fixes** 
fixes #173

**Way to test**:
Run ._/bin/ck8s ops kubectl sc delete pod harbor-harbor-database-0 --namespace harbor_
If the pod restarts with status CrashLoopBackOff, the bug persists. If not, it runs correctly.

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [x] Some test is done to check if the problem still persists.

### CODE REVIEW

**Review by lentzi90 — APPROVED** [review]
LGTM!

File: WIP-CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 ### Fixed
 
-- Elasticsearch slm now deletes excess snapshots also when none of them are older than the maximum age
+- Elasticsearch slm now deletes excess snapshots also when none of them are older than the maximum age.
+- When using harbor together with rook there is a potential bug that appears if the database pod is killed and restarted on a new node. This is fixed by upgrading Harbor to version 1.6.1.
```
Comment (viktor-f) [inline WIP-CHANGELOG.md:8]:
```suggestion
- When using harbor together with rook there is a potential bug that appears if the database pod is killed and restarted on a new node. This is fixed by upgrading the Harbor helm chart to version 1.6.1.
```

**Review by viktor-f — COMMENTED** [review]
So no special migration steps was required for this?

Since this updates both the chart version and the version of Harbor you should add a not in the changelog under "changed" mentioning that we have updated harbor to v2.2.1

File: WIP-CHANGELOG.md (hunk 1 above)
Comment (robinelastisys) [reply WIP-CHANGELOG.md:8]: Nope, nothing needed.

**Review by viktor-f — APPROVED** [review]

**Pull request closed without merging**